modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/H2StatementCache.java

```diff
@@ -102,4 +101,22 @@ void remove(String schemaName, String sql) {
     int size() {
         return lruStmtCache.size();
     }
+
+    /** */
+    public static byte queryFlags(QueryDescriptor qryDesc) {
+        assert qryDesc != null;
+
+        return queryFlags(qryDesc.distributedJoins(), qryDesc.enforceJoinOrder());
+    }
+
+    /** */
+    public static byte queryFlags(
+        boolean distributedJoins,
+        boolean enforceJoinOrder
+        ) {
+        return (byte)(1 +
```

Comment:
why do lack of flags represented by 1, not by 0?

Reply:
Great issue. It's an artifact of the old code. I'll change it.